dashboard/app/controllers/api/v1/users_controller.rb

```diff
@@ -60,7 +60,7 @@ def get_donor_teacher_banner_details
 
   # GET /api/v1/users/<user_id>/school_donor_name
   def get_school_donor_name
-    render json: @user.school_donor_name
```

Comment:
This just preserves existing behavior?

Reply:
Yea, it at least preserves the behavior covered on the tests on `get_school_donor_name` in `users_controller_test`:  https://github.com/code-dot-org/code-dot-org/blob/26060dae2a02568e5d1378dd68011973bbc4c90f/dashboard/test/controllers/api/v1/users_controller_test.rb#L167-L187